symbology/apply-unique-values-with-alternate-symbols/src/main/java/com/esri/samples/apply_unique_values_with_alternate_symbols/ApplyUniqueValuesWithAlternateSymbolsSample.java

```diff
@@ -0,0 +1,231 @@
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // set a viewpoint to the map view centered on a point
+      mapView.setViewpointCenterAsync(vPoint, 25000.0);
+
+      // creating box, button, and label UI components
+      VBox vBox = controlsVBox();
+
+      // resetting map viewpoint on button press
+      btnReset.setOnAction(event -> {
+        // reset the map viewpoint
+        mapView.setViewpointAsync(new Viewpoint(vPoint, 25000.0), 1.5f, AnimationCurve.EASE_IN_OUT_SINE);
+      });
+
+      // updating label scale values
+      mapView.addViewpointChangedListener(event->
+        // update label text
+        lblScale.setText("Current scale: 1:" + String.format("%.2f", mapView.getMapScale())));
+
+      // creating simple markers where the blue square and yellow diamond are alternate symbols
+      // the red triangle is used to create a unique symbol and the purple diamond is the default symbol for the unique value renderer
+      SimpleMarkerSymbol sBlueSquareSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.SQUARE, 0xFF0000FF, 15);
+      SimpleMarkerSymbol sYellowDiamondSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.DIAMOND, 0xFFFFFF00, 15);
+      SimpleMarkerSymbol sRedTriangleSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.TRIANGLE, 0xFFFF0000, 15);
+      SimpleMarkerSymbol sPurpleDiamondSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.DIAMOND, 0xFF800080, 12);
+
+      // converting simpler marker symbols to multilayer point symbols
+      MultilayerPointSymbol mBlueSquareSymbol = sBlueSquareSymbol.toMultilayerSymbol();
+      MultilayerPointSymbol mYellowDiamondSymbol = sYellowDiamondSymbol.toMultilayerSymbol();
+      MultilayerPointSymbol mRedTriangleSymbol = sRedTriangleSymbol.toMultilayerSymbol();
+      MultilayerPointSymbol mPurpleDiamondSymbol = sPurpleDiamondSymbol.toMultilayerSymbol();
+
+      // setting scale range values through reference properties for the multilayer symbols
+      SymbolReferenceProperties blueSquareRef = new SymbolReferenceProperties(10000.0, 5000.0);
+      SymbolReferenceProperties yellowDiamondRef = new SymbolReferenceProperties(20000.0, 10000.0);
+      SymbolReferenceProperties redTriangleRef = new SymbolReferenceProperties(5000.0, 0.0);
+
+      mBlueSquareSymbol.setReferenceProperties(blueSquareRef);
+      mYellowDiamondSymbol.setReferenceProperties(yellowDiamondRef);
+      mRedTriangleSymbol.setReferenceProperties(redTriangleRef);
+
+      // Creating unique value for the red triangle and list of alternate symbols
+      UniqueValueRenderer.UniqueValue uniqueValue =
+        new UniqueValueRenderer.UniqueValue("unique values based on request type", "unique value",
+          mRedTriangleSymbol, List.of("Damaged Property"), List.of(mBlueSquareSymbol, mYellowDiamondSymbol));
+
+      // Creating unique value renderer
+      UniqueValueRenderer uniqueValRenderer = new UniqueValueRenderer();
```

Comment:
we can use var here with updated variable name

```suggestion
      var uniqueValueRenderer = new UniqueValueRenderer();
```

We could also make use of the second constructor here for UniqueValueRenderer (https://developers.arcgis.com/java/api-reference/reference/com/esri/arcgisruntime/symbology/UniqueValueRenderer.html): the empty one above creates default empty lists/strings which we then add to later in the current code, but we can pass those properties directly in for the second constructur (using an empty string as the label). This would cut down on a couple lines of code without affecting readability

```java
      UniqueValueRenderer uniqueValRenderer = new UniqueValueRenderer(List.of("req_type"), List.of(uniqueValue), "", mPurpleDiamondSymbol);
```